library/src/main/scala/scala/scalajs/runtime/WasPublicBeforeTyper.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalajs.runtime
+
+import scala.annotation.StaticAnnotation
+
+class WasPublicBeforeTyper extends StaticAnnotation
```

Comment:
Also, this should not be static: We don't want this to be retained in `.class` files.